<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppTools::Internal::CppFileSettingsPage</class>
 <widget class="QWidget" name="CppTools::Internal::CppFileSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>142</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="headerSuffixLabel">
     <property name="text">
      <string>Header suffix:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="headerSuffixComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sourceSuffixLabel">
     <property name="text">
      <string>Source suffix:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="sourceSuffixComboBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="lowerCaseFileNamesCheckBox">
     <property name="text">
      <string>Lower case file names</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="licenseTemplateLabel">
     <property name="text">
      <string>License template:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Utils::PathChooser" name="licenseTemplatePathChooser" native="true"/>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>licenseTemplatePathChooser</zorder>
  <zorder>sourceSuffixLabel</zorder>
  <zorder>headerSuffixLabel</zorder>
  <zorder>lowerCaseFileNamesCheckBox</zorder>
  <zorder>headerSuffixComboBox</zorder>
  <zorder>sourceSuffixComboBox</zorder>
  <zorder>licenseTemplateLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
